searx/templates/simple/preferences/cookies.html

```diff
@@ -40,7 +40,7 @@ <h4>
   </pre>{{- '' -}}
 </div>{{- '' -}}
 <p class="small_font">
-  {{- _('Specifying custom settings in the preferences URL can be used to sync preferences across devices.') -}}
+  {{- _('A URL with preference settings allows you to sync your preferences across devices.') -}}
```

Comment:
```suggestion
  {{- _('A URL containing your preferences. This URL can be used to restore your settings on a different device.') -}}
```
would be my take since "preferences" is often used as a synonym to "settings", that'd almost be twice the same word.